.travis.yml

```diff
@@ -6,6 +6,9 @@ matrix:
     - python: 3.4
     - python: 3.5
     - python: 3.6
+    - python: 3.7
+      dist: xenial
```

Comment:
Why specify ``dist`` and ``sudo`` for 3.7 when we aren't doing that for other Python versions?

Reply:
Because Python 3.7 does not work without those settings on Travis CI: https://github.com/travis-ci/travis-ci/issues/9815

(Something about new OpenSSL version requirement that means 3.7 cannot be built on Ubuntu trusty, which is the default Travis CI image AFAIU.  No clue why it fails without the `sudo` bit, but that issue indicates that it's also required.)